Framework/Core/include/Framework/DataAllocator.h

```diff
@@ -42,7 +43,7 @@ namespace framework {
 class DataAllocator
 {
 public:
-  using AllowedOutputsMap = std::vector<OutputRoute>;
+  using AllowedOutputRoutes = std::vector<OutputRoute>;
```

Comment:
try indenting `public` by one space. I think `clang-format` gets confused, because that line is not indented, so it want's all following line to be indented by only one space. Same for `private` below.